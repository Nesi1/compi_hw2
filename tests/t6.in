void main(){
    if ( true and false or true and false)
        return 0;
}