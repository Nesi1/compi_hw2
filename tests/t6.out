6: RetType ->  VOID
7: Formals -> epsilon
41: Exp -> TRUE
42: Exp -> FALSE
44: Exp -> Exp AND Exp
41: Exp -> TRUE
42: Exp -> FALSE
44: Exp -> Exp AND Exp
45: Exp -> Exp OR Exp
38: Exp -> NUM
21: Statement -> RETURN Exp SC
22: Statement -> IF LPAREN Exp RPAREN Statement
12: Statements -> Statement
4: FuncDecl -> RetType ID LPAREN Formals RPAREN LBRACE Statements RBRACE
2: Funcs -> epsilon
3: Funcs -> FuncDecl Funcs
1: Program -> Funcs
